gen/go/v1/config.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.27.1
-// 	protoc        v3.18.0
+// 	protoc-gen-go v1.26.0
```

Comment:
Ugh is this a cause for concern? Should be ok I think as it's not a major difference?